src/plugins/wizard/public/application/components/side_nav.scss

```diff
@@ -25,6 +46,5 @@
 }
 
 .wizDatasourceSelect {
-  max-width: $wizSideNavWidth;
-  padding: $euiSize $euiSize 0 $euiSize;
+  max-width: calc(#{$wizSideNavWidth} - 1px);
```

Comment:
Should we track this as an enhancement we need in OUI?

Reply:
This is a specific with we need for the side panels. I've opened an issue to add resizing and collapsing https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2241. That should move this from CSS to JS